packages/components/src/stories/SquigglePlayground.stories.tsx

```diff
@@ -76,8 +77,19 @@ export const WithExtraControls: Story = {
   args: {
     defaultCode: "normal(5,2)",
     height: undefined,
-    renderExtraControls: () => (
-      <button className="bg-green-500 px-2">Extra button</button>
+    renderExtraControls: ({ openModal }) => (
```

Comment:
I can't seem to open it, in the Storybook playground.
<img width="1778" alt="image" src="https://github.com/quantified-uncertainty/squiggle/assets/377065/99aecd72-d850-4742-bc91-07e8395b011a">
https://squiggle-components-6y7ar47h3-quantified-uncertainty.vercel.app/?path=/story/squiggle-squiggleplayground--with-extra-controls

Reply:
That's an old build, try here: https://squiggle-components-git-playgroun-d4dac0-quantified-uncertainty.vercel.app/?path=/story/squiggle-squiggleplayground--with-extra-controls

Reply:
Ah cool, thanks!